src/components/docs/Search.tsx

```diff
@@ -0,0 +1,24 @@
+import "@docsearch/css";
+import { DocSearch } from "@docsearch/react";
+import "../../styles/global.css";
+const Search = () => {
+  return (
+    <div className=" lg:w-[15rem] xl:w-[18rem]">
+      <DocSearch
```

Comment:
Renders lots of "duplicate" results that are hard to know which to click on. When I search "gas service" do I want "Axelar Documentation" or "Axelar Documentation"?

![image](https://github.com/user-attachments/assets/9d01092f-aabe-4f36-be33-3c0a5ebca022)